src/lib/components/forms/form/helper/terra-form-field.helper.ts

```diff
@@ -221,7 +221,8 @@ export class TerraFormFieldHelper {
         let max: number;
 
         if (isString(range)) {
-            let match: RegExpExecArray = /^\[(\d*),(\d*)]$/.exec(range as string);
+            const trimmedRange: string = (range as string).replace(/\s/g, '');
+            let match: RegExpExecArray = /^\[(\d*)\s*,\s*(\d*)]$/.exec(trimmedRange as string);
```

Comment:
Theoretisch könntest du die RexExp noch so erweitern dass durch das "trimmen" zuvor garnicht mehr brauchst, oder?
```suggestion
            let match: RegExpExecArray = /^\[\s*(\d*)\s*,\s*(\d*)\s*]$/.exec(trimmedRange as string);
```